openslides_backend/action/relations/user_committee_calculate_handler.py

```diff
@@ -79,7 +79,11 @@ def process_field(
         )
         new_committees_ids.update(committee_ids)
         added_ids = new_committees_ids - db_committee_ids
-        removed_ids = db_committee_ids - new_committees_ids
+        removed_ids = (
+            db_committee_ids - new_committees_ids
+            if action not in ("meeting.import", "meeting.clone")
+            else []
```

Comment:
Yes, that works and seems to be correct, but I would add this kind of code in `meeting.import`.